CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 0.2.5
+
+### Fixed
+
+- Changed number of color pairs in alacritty terminfo file from 65,536 to 32,767 to fix color issues in certain ncurses-based programs.
```

Comment:
Please follow the style of the other messages in the change log. This includes removing the punctuation at the end and trimming the line length to 80-100 characters max.

Also, beginning a `Fixed` message with `Changed...` is probably sending the wrong signal. If something has been fixed, it should be described what has been fixed, rather than what has been changed. So changing it to `Fix color issues in ncurses programs by ...` would be more appropriate.

Reply:
I've made this update. Let me know if I captured everything.